hydromt_fiat/workflows/exposure_vector.py

```diff
@@ -399,6 +411,9 @@ def setup_asset_locations(self, asset_locations: str) -> None:
 
         # Set the asset locations to the geometry variable (self.exposure_geoms)
         # and set the geom name
+        if len(assets.columns) > 2:
```

Comment:
why not always do that?

Reply:
?

Reply:
That way a user can supply the same file for occupancy and assets. Instead of dividing already a BF asset file incl all occupncy into 2 fies (extra manual step) the user can use the same file but then here just the Object ID and geoms are taken